config/initializers/postcode_regex.rb

```diff
@@ -0,0 +1 @@
+POSTCODE_REGEXP = /^(([A-Z]{1,2}[0-9][A-Z0-9]?|ASCN|STHL|TDCU|BBND|[BFS]IQQ|PCRN|TKCA) ?[0-9][A-Z]{2}|BFPO ?[0-9]{1,4}|(KY[0-9]|MSR|VG|AI)[ -]?[0-9]{4}|[A-Z]{2} ?[0-9]{2}|GE ?CX|GIR ?0A{2}|SAN ?TA1)$/i
```

Comment:
Would have added this to the PostcodeService I think just in terms of bundling together related functionality but I suppose this matches the pattern we use for dates.